src/Routes/ImageManager/steps/registration.js

```diff
@@ -33,7 +33,10 @@ export default {
     {
       component: 'ssh-input-field',
       name: 'credentials',
-      validate: [{ type: validatorTypes.REQUIRED }],
+      validate: [
+        { type: validatorTypes.REQUIRED },
+        { type: validatorTypes.PATTERN, pattern: /^(ssh-[a-z]{3}) \S+/ },
```

Comment:
There can be either:
* `ssh-rsa` - for RSA keys
* `ssh-dss` - for DSA keys
* `ecdsa-sha2-nistp256 ` - for ECDSA keys, where the number behind `nistp` is either 256, 384 and 521